esm-samples/jsapi-create-react-app/package.json

```diff
@@ -9,7 +9,7 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-scripts": "^4.0.3",
+    "react-scripts": "5.0.0",
```

Comment:
Should this be `^5.0.0` for consistency?

Reply:
👍